cli/cmd/cmd.go

```diff
@@ -206,6 +206,8 @@ func (c *ServicedCli) cmdInit(ctx *cli.Context) error {
 		fmt.Fprintf(os.Stderr, "Unable to set logging options: %s\n", err)
 	}
 
+	isvcs.InitAllIsvcs()
```

Comment:
The issue with this request is If I do not call it after `config.LoadOptions(options)`, the configs are not loaded which is done one line up.  If I don't call it before `if err := setIsvcsEnv(ctx); err != nil {`, I get a panic because the objects have not been created.  Lets discuss first thing monday and I can demo for you.